src/components/elements/__tests__/Table.test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { StyleSheetTestUtils } from 'aphrodite'
+import { render } from 'enzyme'
+import { Table } from '..'
```

Comment:
for tests, we should avoid importing and evaluating extra code, so import statements in all tests should be made this way: ```import Table from '../Table'```